src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -89,11 +89,33 @@ bool GetInteriorVehicleDataRequest::ProcessCapabilities() {
   return true;
 }
 
+void GetInteriorVehicleDataRequest::FilterDisabledModuleData(
+    smart_objects::SmartObject& module_data) {
+  // If radioEnable is false, remove all other radio parameters from the
+  // message.
+  if (module_data.keyExists(message_params::kRadioEnable) &&
+      module_data[message_params::kRadioEnable] == false) {
+    for (auto data = module_data.map_begin(); data != module_data.map_end();
+         ++data) {
+      if (data->first != message_params::kRadioEnable) {
+        module_data.erase(data->first);
```

Comment:
@JackLivio wont the iterator invalidate after erazing from map? Can we use something like std::remove?